internal/testdir/testdir.go

```diff
@@ -42,8 +38,11 @@ const goMod = `
 	)
 `
 
-func New() *Dir {
+func New(dir string) *Dir {
 	return &Dir{
+		dir:         dir,
+		Backup:      false, // TODO: re-enable and store snapshots in ./bud/tmp
```

Comment:
Snapshotting and restoring test directories have been turned off by default. I want to re-enable this again when there's more tests (like the one above) and it's hooked into CI.